content/01-install/01a-install-client-windows.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Install Client on Windows
+metaTitle: Install Client on Windows
+---
+
+This section outlines how to install the Mantis client on your Windows machine.
+
+Follow these instructions to install the Mantis client or go [here for instructions to install the Mantis wallet](/install/install-wallet).
```

Comment:
looks like a broken link to me, is on on purpose?

Reply:
When I run the Gatsby develop environment it is not broken. I am not familiar with this environment, so maybe I missed something when I checked the file in.